tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -665,58 +679,26 @@ def _fill_indices(indice_value):
         return indices
     np_indices = _fill_indices(indice_value)
 
-    compare_tf_with_tvm([np_data, np_indices], ['in_data:0', 'indices:0'], 'GatherV2:0')
+    if tf.__version__ < LooseVersion('1.7'):
+        output_name = 'Gather:0'
+    else:
+        output_name = 'GatherV2:0'
+    compare_tf_with_tvm([np_data, np_indices], ['in_data:0', 'indices:0'], output_name)
```

Comment:
Why not use this method to get output_name, I guess it works on both versions
```
out = tf.gather(in_data, indices, axis=axis)
output_name = out.name
```

Reply:
done